app/views/restaurants/index.html.haml

```diff
@@ -1,14 +1,33 @@
-%h1 Ufoods
-%p What do you feel like tonight?
-- @categories.each do |category|
-  .temp_box
-    %h3
-      = category.name
-    %p
-      = category.description
-  .temp_box2
-    - category.restaurants.near(@user_location, 3, units: :km).each do |restaurant|
-      = link_to restaurant.name, restaurant_path(restaurant)
-      %p
-        = restaurant.address
-#map{style: 'width: 800px; height: 400px', data: {test: {env: "#{Rails.env.test?}"}}}
+.container
+  #categories
+    %h1
+      What do you feel like tonight?
+      = image_tag 'wheat', height: '200', width: '100%'
```

Comment:
I would suggest you use `style` and use css (the best solution is to define a css class) Setting the height and with on a tag is sooo nineties ;-)